source/website/package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "private": true,
   "scripts": {
-    "serve": "vue-cli-service serve",
+    "serve": "NODE_OPTIONS=--openssl-legacy-provider vue-cli-service serve",
```

Comment:
What error does this resolve?

Reply:
I was researching the build pipeline error and some resources suggested this since the Vue build was failing - but I just tested it and I think it was due to the Node version Immanuel found, so I ended up reverting this